.github/workflows/e2e-full.yml

```diff
@@ -18,14 +18,15 @@ jobs:
       matrix:
         globalnet: ['', 'globalnet']
         lighthouse: ['', 'lighthouse']
+        ovn: ['', 'ovn']
```

Comment:
This looks fine, but as it doubles the number of jobs, can we add two additional jobs using the [include](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs#expanding-or-adding-matrix-configurations) options and validate the following additional jobs?
1. GN + LH + OVN
2. LH + OVN

Reply:
Note that this commit is now part of https://github.com/submariner-io/subctl/pull/883. This PR adds 4 more jobs, so you're saying you want to exclude these?

1. OVN
2. OVN + GN